README.md

```diff
@@ -19,28 +21,31 @@
 Wing enables developers to build distributed systems that leverage cloud services as first-class citizens by combining infrastructure ***and*** application code in a safe and unified programming model (aka "*cloud-oriented*").
 Wing programs can be executed locally (*yes, no internet required*) using a fully-functional simulator, or deployed to any cloud provider (*yes, Wing programs are portable across providers*).
 
-Our mission is to bring back your creative flow and close the gap between imagination and creation. 
+The mission of Winglang is to bring back your creative flow and close the gap between imagination and creation.
 
 Developing for the cloud today requires mastering various layers of cloud stack, IAM roles, networking, and numerous tools, along with finding creative ways to test and debug code. In addition, long deployment times hinder iteration cycles and take developers out of their creative flow. 
 
-We fulfill our mission by reducing cognitive load and increasing iteration speeds. How? by enabling you to work at a higher level of abstraction, letting you focus on business logic instead of cloud mechanics, and write much less code. We also provide you with a set of tools that let you test your code locally, significantly faster than before.
+Winglang addresses these pains by letting you to work at a higher level of abstraction, and allowing you focus on business logic instead of cloud mechanics, only surfacing low-level details when its needed.
```

Comment:
```suggestion
Winglang addresses these pains by letting you work at a higher level of abstraction and allowing you to focus on business logic instead of cloud mechanics, only surfacing low-level details when its needed.
```